ErrorReporting/src/Bootstrap.php

```diff
@@ -232,4 +242,22 @@ public static function shutdownHandler()
             }
         }
     }
+
+    public static function getFunctionNameForReport($trace = [])
+    {
+        if (null === $trace) {
```

Comment:
This could probably be `empty($trace)` then you wouldn't have to explicitly pass in null and this method could always accept an array.

Reply:
The reason I didn't do this is I want to tell the difference between when we don't HAVE a trace and when the trace is empty. When the trace is empty, it's because the error happened outside of a function (e.g. '<?php throw Exception('')') but when we don't have a trace, it's because that particular error cannot detect a function (e.g. a syntax error).

I can change this to be more obvious, and add documentation to explain.